environments.json

```diff
@@ -1265,7 +1265,7 @@
       "non-standard"
     ]
   },
-  "safari51": {
```

Comment:
@ljharb I can wait for the @afmenez's patch to be landed & then rebase mine. I'm fine with solving those merge conflicts, I just hope I won't have to do that multiple times.